docker-compose.yml

```diff
@@ -1,25 +1,30 @@
 version: '3'
 services:
-  db:
-    image: postgis/postgis
+  postgres:
+    image: postgis/postgis:latest
     environment:
-      POSTGRES_DB: rgd
+      POSTGRES_DB: django
       POSTGRES_PASSWORD: postgres
+    ports:
+      - ${DOCKER_POSTGRES_PORT-5432}:5432
     volumes:
-      - "dbdata:/var/lib/postgresql/data"
+      - dbdata:/var/lib/postgresql/data
 
   rabbitmq:
     image: rabbitmq:management
+    ports:
+      - ${DOCKER_RABBITMQ_PORT-5672}:5672
+    volumes:
+    - mqdata:/var/lib/rabbitmq
```

Comment:
Remove this, we never want to persist the state of a RabbitMQ volume across container re-creation.